dcos/dcos-ui
# DCOS-43091: [COPS] Logs view / Nodes view don't scroll in Firefox

Allow the logs to be scrollable in Firefox.

Closes DCOS-43091

## Testing
1. Test in both Firefox and Chrome
2. Start a service
3. Go to Services Tab, click the service.
4. Click a task
5. Open the logs tab
6. Verify that Error(stderr) and Output(stdout) are scrollable
7. Go to Nodes
8. Click a node
9. Click a task
10. Open the logs tab
11. Verify that Error(stderr) and Output(stdout) are scrollable

## Trade-offs
Firefox need an explicit height (so not `100%` or `auto`) for `overflow: auto` or `overflow: scroll` to work.

## Dependencies
None.

## Screenshots

### Before
![screenshot from 2018-10-05 17-35-33](https://user-images.githubusercontent.com/40791275/49650706-d21b1100-fa35-11e8-83a0-934269e0b693.png)

### After
![peek 2018-12-07 15-24](https://user-images.githubusercontent.com/40791275/49650664-b152bb80-fa35-11e8-929d-f1c7ecbdf3f2.gif)
